libsolidity/interface/CompilerStack.cpp

```diff
@@ -1645,34 +1643,35 @@ bytes CompilerStack::createCBORMetadata(Contract const& _contract, bool _forIR)
 namespace
 {
 
-Json::Value gasToJson(GasEstimator::GasConsumption const& _gas)
+Json gasToJson(GasEstimator::GasConsumption const& _gas)
 {
 	if (_gas.isInfinite)
-		return Json::Value("infinite");
+		return Json("infinite");
 	else
-		return Json::Value(util::toString(_gas.value));
+		return Json(util::toString(_gas.value));
+//		return Json(_gas.value.str());
```

Comment:
Commented-out code.

Reply:
I think this meant to be kept as a TODO to double check stuff. Will replace it with an actual TODO.